apertium-eng-spa.eng-spa.lrx

```diff
@@ -8715,7 +8715,185 @@ convertirse en un personaje de moda o importante. -->
           <match lemma="save" tags="vblex.*"><select lemma="ahorrar" tags="vblex.*"/></match>
           <match lemma="money" tags="n.*"/>
         </rule>
-
-
+	
+    <!--
+      eng: as part of
+      spa: como parte de
+    -->
+    <rule weight="1.0">
+      <match lemma="as" tags="*">
+        <select lemma="como" tags="cnjadv"/>
+      </match>
+      <match lemma="part" tags="n.*"/>
+      <match lemma="of" tags="pr"/>
+    </rule>
+
+    <!--
+      eng: the country's status
+      spa: el estatus del país
+    -->
+    <rule weight="1.0">
+      <or>
+        <match lemma="country" tags="n.*"/>
+        <match lemma="nation" tags="n.*"/>
+        <match lemma="state" tags="n.*"/>
+      </or>
+      <match lemma="'s" tags="gen"/>
+      <match lemma="status" tags="n.*">
+        <select lemma="estatus" tags="n.*"/>
+      </match>
+    </rule>
+
+    <!--
+      eng: global military power
+      spa: potencia militar global
+    -->
+    <rule weight="1.0">
+      <or>
+        <match lemma="global" tags="adj"/>
+        <match lemma="national" tags="adj"/>
+        <match lemma="continental" tags="adj"/>
+        <match lemma="international" tags="adj"/>
+        <match lemma="regional" tags="adj"/>
+      </or>
+      <match lemma="military" tags="adj"/>
+      <match lemma="power" tags="n.*">
+        <select lemma="potencia" tags="n.*"/>
+      </match>
+    </rule>
+
+    <!--
+      eng: socioeconomic performance
+      spa: rendimiento socioeconómico
+    -->
+    <rule weight="1.0">
+      <or>
+        <match lemma="socioeconomic" tags="adj"/>
```

Comment:
"socioeconomic performance"?

Reply:
I found that in the Wikipedia page:

"States continues to rank very high in measures of socioeconomic performance, including average wage, median income, median wealth, human development, per capita GDP, and worker productivity."

Reply:
Good example.